### PR TITLE
Add reviews to DB and display them in the course page

### PR DESCRIPTION
**Pull request by theborgh**
- Added new review mongoDB schema
- Added /newReview POST endpoint
- Display all reviews for the given course (basic styles, star icons not yet implemented)

### CODE REVIEW

**Comment by Noahliberty** [comment]
Aprroved.

File: app.js [hunk 1]
```diff
@@ -43,23 +43,29 @@ const courseSchema = new mongoose.Schema({
...
-	review: String,
 	price: Number,
 	isFree: Boolean,
-	reviewTitle: String,
-	reviewDetails: String
 });
 
 const Course = mongoose.model("Course", courseSchema);
 
+const reviewSchema = new mongoose.Schema({
```
Comment (vansi42) [inline app.js:52]: No need to state "id" as a field, it is automatically generated :)

**Review by vansi42 — COMMENTED** [review]
Looks great!

File: app.js (hunk 1 above)
Comment (theborgh) [reply app.js:52]: Oh cool, thanks for telling me!

**Pull request merged**